pennylane_lightning/core/src/bindings/Bindings.hpp

```diff
@@ -769,23 +769,152 @@ void registerLightningTensorBackendAgnosticMeasurements(PyClass &pyclass) {
         });
 }
 
+/**
+ * @brief Register observable classes.
+ *
+ * @tparam LightningBackendT
+ * @param m Pybind module
+ */
+template <class LightningBackendT>
+void registerBackendAgnosticObservablesTensor(py::module_ &m,
```

Comment:
Would it make sense to call these `registerTensorObservables`?

Reply:
I will keep the current naming to continue with the other devices style. :sweat_smile: